src/js/components/ConfigurationView.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import mixin from 'reactjs-mixin';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import DCOSStore from '../stores/DCOSStore'
+import DescriptionList from './DescriptionList';
+import Service from '../structs/Service';
+
+class ConfigurationView extends mixin(StoreMixin) {
+  constructor() {
+    super();
+
+    this.store_listeners = [{name: 'dcos', events: ['change']}];
+  }
+
+  render() {
+    let {headline, service, versionID} = this.props;
+    let configuration = service.getVersions().get(versionID);
+
+    if (!configuration) {
+      DCOSStore.fetchServiceVersion(service.getId(), versionID);
```

Comment:
Oh, yeah! This is a WIP implementation, as I wasn't sure how `componentWillMount` will work in combination with the `StoreMixin`. I will definitely refactor this!